### PR TITLE
correctly define route for homepage controller

### LINKED ISSUE #762: make-homepage.rst
http://symfony.com/doc/current/cmf/tutorial/make-homepage.html

The @Route annotation doesn't work if this piece of code is not in @Bundle/Config/routing.yml
```
default:
    resource: "@DewittCmsBundle/Controller/DefaultController.php"
    type:     annotation
    prefix:   /
```

### PR DESCRIPTION
**Pull request by dbu**
fix #762

### CODE REVIEW

**Comment by WouterJ** [comment]
I think we should use annotations instead, as it's the official Symfony best practice. Also, if I understand the issue correctly, it's more about the `make_homepage` route.

**Comment by dbu** [comment]
looking at http://symfony.com/doc/current/bundles/SensioFrameworkExtraBundle/annotations/routing.html it seems that annotations don't work without adding the controller to the routing file. but indeed we have more cases of annotations on controllers in this tutorial, at least we have to be consistent.

what is the best practice to handle the need to add the controller to the routing file? mention it at the first place where we use the controller, and i update the other code samples to make clear that i am adding new methods to the same controller?

**Comment by WouterJ** [comment]
Well, if we rename the bundle to just use `AppBundle`, things work automatic. Otherwise, mention it in the beginning to add:
```yaml
_my_bundle:
    resource: '@MyBundle/Controller'
    type: annotation
```

**Comment by dbu** [comment]
AppBundle would be best practice. so i guess i will change the
namespaces, ok?

**Comment by dbu** [comment]
i updated the tutorial to use AppBundle instead of Acme\CmsBundle and use the route annotation on the controller.

File: tutorial/getting-started.rst [hunk 1]
```diff
@@ -23,27 +23,16 @@ Each part of the tutorial will detail the packages that it requires (if any) in
...
-            "symfony-cmf/routing-bundle": "~1.3",
-            "symfony-cmf/routing": "~1.3"
-        },
-        ...
-    }
+.. code-block:: bash
 
-Note that each time you modify your ``composer.json`` file you are required to
-run ``composer update``.
+    composer require symfony-cmf/routing-auto-bundle \
```
Comment (wouterj) [inline tutorial/getting-started.rst]: please prefix this with `$`

**Comment by wouterj** [comment]
very nice job! Feel free to merge

**Pull request merged**